tests/unit_tests/test_load_callbacks.py

```diff
@@ -1,58 +1,117 @@
+from unittest.mock import patch
+
 import dash_bootstrap_components as dbc
+import plotly.graph_objects as go
+import pytest
 from dash import html
 from dash._callback_context import context_value
 from dash._utils import AttributeDict
+from eitprocessing.datahandling.eitdata import Vendor
+from eitprocessing.datahandling.sequence import Sequence
 
 import eit_dash.definitions.element_ids as ids
-from eit_dash.callbacks.load_callbacks import load_file, open_modal, show_info
-
-
-def test_show_info_callback():
-    output = show_info(1, None, 0)
-
-    mock_data_card = [
-        dbc.Card(
-            dbc.CardBody(
-                [
-                    html.H4("Dataset 1", className="card-title"),
-                    html.H6("Timpel", className="card-subtitle"),
-                    dbc.Row("N_signals: 3", style={"margin-left": 10}),
-                    dbc.Row("duration: 12", style={"margin-left": 10}),
-                    dbc.Row("n_frames: 234", style={"margin-left": 10}),
-                    dbc.Row("filename: file.bin", style={"margin-left": 10}),
-                    dbc.Row("etc: etc", style={"margin-left": 10}),
-                ],
-            ),
-            id="card-1",
+import eit_dash.definitions.layout_styles as styles
+from eit_dash.callbacks.load_callbacks import load_selected_data, show_info
+from eit_dash.definitions.option_lists import InputFiletypes
+from tests.conftest import data_path
+
+expected_continuous_data = [
+    {"label": "airway pressure", "value": 0},
+    {"label": "flow", "value": 1},
+    {"label": "volume", "value": 2},
+    {"label": "CO2", "value": 3},
+]
+
+first_sample = 100
+last_sample = 1000
+
+cut_file_length = last_sample - first_sample
```

Comment:
Shouldn't these be capitalized by convention?

Reply:
absolutely! I will change this